src/world/conquest_system.cpp

```diff
@@ -62,8 +76,11 @@ bool ConquestSystem::handleMessage(std::vector<uint8> payload,
         uint64 ipp = from_addr.s_addr;
         ipp |= (((uint64)from_port) << 32);
 
-        // Send influence data to the requesting map server
-        sendInfluencesMsg(true, ipp);
+        submit([this, ipp]()
+               {
```

Comment:
I appreciate that clang-format probably did this for you, and that this is a more common layout in the Mac dev space, but we tend to wrap all lambda's in `clang-format off` to avoid this broken indentation: https://github.com/LandSandBoat/server/wiki/Development-Guide#Lambdas

Reply:
oh ya I had applied linter but did not even see the style.. I actually had it as you describe before. Looks so odd otherwise.